src/modules/esl-event-listener/docs/2-public-api.md

```diff
@@ -0,0 +1,266 @@
+## Public API (`ESLEventUtils`)
+
+The units mentioned earlier are mostly implementation details of the module.
+
+`ESLEventUtils` is a facade for all ESL event listener module capabilities.
+
+Here is the module Public API:
+
+<a name="-esleventutilssubscribe"></a>
+
+### ⚡ `ESLEventUtils.subscribe`
+
+Creates and subscribes an `ESLEventListener`.
```

Comment:
```suggestion
Creates and subscribes to an `ESLEventListener`.
```

Reply:
We subscribe the listener but not to the listener